posthog/settings.py

```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
     # Add hard-coded feature flags for static self-hosted releases here
     "5440-multivariate-support",
     "new-paths-ui-edge-weights",
+    "sigma-analysis",
+    "new-sessions-player",
+    "new-sessions-player-events-list",
+    "938-breakdown-by-multiple-properties",
```

Comment:
@pauldambra do we want to include in the release yet?

Reply:
Ah, not yet... Need to roll it out to beta users on cloud still :/